apps/web/app/lib/survey-builder.ts

```diff
@@ -393,7 +402,8 @@ export const buildSurvey = (
     channels: ("link" | "app" | "website")[];
     role: TTemplateRole;
     description: string;
-    questions: TSurveyQuestion[];
+    questions?: TSurveyQuestion[];
```

Comment:
I think we can remove `questions` from here and also in general remove the question types and all the older code from this file as well. It should be fine to only support blocks now since after the migration everything will be blocks anyway. Removing the older types would become easier as well since they'll be no longer used anywhere